modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/add_to_cart/add_to_cart.scss

```diff
@@ -104,3 +104,9 @@
 		}
 	}
 }
+
+.minium-frame {
```

Comment:
Shouldn't this be in the theme CSS instead since it's for minium?

Reply:
Moved on minium theme